app/Http/Controllers/ContestController.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Gladiator\Http\Controllers;
+
+use Gladiator\Models\Contest;
+use Gladiator\Http\Requests\ContestRequest;
+
+class ContestController extends Controller
```

Comment:
yeah I see what you mean, some stuff ends up sounding funny, but I'd rather follow the expected standard. I'll be expecting it to be plural when I use it and get errors :confounded: